hdinsight-linux-add-hue-on-existing-cluster/azuredeploy.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
I suggest you reconsider the name for this tempalte. hdinsight-linux-hue-existing-cluster is better and will fit in the Github column limits

![image](https://cloud.githubusercontent.com/assets/6164178/16703081/7c278684-451f-11e6-9a60-6cf9a71664bb.png)